04_setup_ironic.sh

```diff
@@ -71,6 +71,8 @@ if [ ! -z "${MIRROR_IMAGES}" ]; then
         --from ${OPENSHIFT_RELEASE_IMAGE} \
         --to-release-image ${LOCAL_REGISTRY_DNS_NAME}:${LOCAL_REGISTRY_PORT}/localimages/local-release-image:${OPENSHIFT_RELEASE_TAG} \
         --to ${LOCAL_REGISTRY_DNS_NAME}:${LOCAL_REGISTRY_PORT}/localimages/local-release-image 2>&1 | tee ${MIRROR_LOG_FILE}
+    echo "export MIRRORED_RELEASE_IMAGE=$OPENSHIFT_RELEASE_IMAGE" > mirrored_release_image
+    _tmpfiles="$_tmpfiles mirrored_release_image"
```

Comment:
Oh shoot, I just realized this doesn't work. _tmpfiles are cleaned up on the exit of the script, so they won't persist to the next one.